packages/atomic/src/components/recommendations/atomic-recs-result/atomic-recs-result.tsx

```diff
@@ -37,6 +39,13 @@ export class AtomicRecsResult {
    */
   @Prop() stopPropagation?: boolean;
 
+  /**
+   * The result link to use when the result is clicked in a grid layout.
+   *
+   * @default - An `atomic-result-link` without any customization.
+   */
+  @Prop() linkContent: ParentNode = new DocumentFragment();
+
```

Comment:
When would you want to use something else than the result-link ?

Reply:
Not sure exactly, I just mirrored what is in atomic-result and atomic-product. @louis-bompart maybe you can provide some insights?

Reply:
Only a result-link will be considered, but, it can be customized with its attribute and option.
Like:
```
<atomic-result-template>
  <template slot="link">
    <atomic-result-link href-template="potato">
      <a slot="attributes" target="_blank"></a>
    </atomic-result-link>
  </template>
</atomic-result-template>
```

Reply:
ah I see this is just mirrored already